packages/editor-ui/src/Interface.ts

```diff
@@ -184,6 +184,7 @@ export interface IRestApi {
 	retryExecution(id: string, loadWorkflow?: boolean): Promise<boolean>;
 	getTimezones(): Promise<IDataObject>;
 	getBinaryBufferString(dataPath: string): Promise<string>;
+	getScopes: (credentialType: string) => Promise<string[]>;
```

Comment:
no, we never never add to IRestAPi... can you move this to store and add `deprecated` on top of this interface and warning not to add to it